inline.go

```diff
@@ -638,23 +638,33 @@ func stripMailto(link []byte) []byte {
 	}
 }
 
+// autoLinkType specifies a kind of autolink that gets detected.
+type autoLinkType int
+
+// These are the possible flag values for the autolink renderer.
+// Only a single one of these values will be used; they are not ORed together.
+const (
+	notAutolink autoLinkType = iota
```

Comment:
Seeing an inconsistency here. Is it one word like `Autolink`/`autolink` or two words like `AutoLink`/`autoLink`?

I'd suggest one word like `Autolink`/`autolink` and use that everywhere.